src/components/KBadge/KBadge.cy.ts

```diff
@@ -20,6 +20,21 @@ describe('KBadge', () => {
   // Loop through appearances
   Object.keys(appearances).map(a => rendersCorrectAppearance(a))
 
+  it('renders with borders', () => {
```

Comment:
Can you also add a negative test, as in, the default badge does **not** have the border?